api/src/main/java/uk/gov/dvsa/mot/trade/api/RfrAndAdvisoryItem.java

```diff
@@ -30,4 +31,15 @@ public RfrAndAdvisoryItem setType(String type) {
         this.type = type;
         return this;
     }
+
+    public Boolean getDangerous() {
+
+        return dangerous;
+    }
+
+    public RfrAndAdvisoryItem setDangerous(Boolean dangerous) {
```

Comment:
Again, do you need to use the Boolean class instead of bool primitive?

Reply:
AFAIK it is field which is coming from DB and is nullable

Reply:
OK, but should it be nullable? Surely a defect is either dangerous or not, it can't be neither dangerous nor not dangerous.

Perhaps this column should have a default value of false?